Develappers.BillomatNet/Helpers/ArticleMappingExtensions.cs

```diff
@@ -177,5 +177,35 @@ internal static Types.PagedList<ArticleTag> ToDomain(this ArticleTagList value)
                 List = value.List?.Select(x => x.ToDomain()).ToList()
             };
         }
+
+        internal static Api.Article ToApi(this Article value)
+        {
+            if (value == null)
+            {
+                return null;
+            }
+            return new Api.Article
+            {
+                Id = value.Id.ToString(),
+                Created = value.Created.ToApiDate(),
+                ArticleNumber = value.ArticleNumber,
+                CurrencyCode = value.CurrencyCode,
+                Description = value.Description,
+                Number = value.Number.ToString(),
+                NumberLength = value.NumberLength,
+                NumberPre = value.NumberPre,
+                PurchasePrice = value.PurchasePrice.ToString(),
+                PurchasePriceNetGross = value.PurchasePriceNetGross,
+                SalesPrice = value.SalesPrice.ToString(),
```

Comment:
bitte das mal mit unterschiedlichen Thread-Cultures testen. Ich befürchte, dass das in einem deutschen Thread fehlschlägt